fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -6238,11 +6331,50 @@ Future<Version> FileBackupAgent::restore(Database cx,
 	                                    unlockDB,
 	                                    onlyApplyMutationLogs,
 	                                    inconsistentSnapshotOnly,
-	                                    beginVersion,
 	                                    encryptionKeyFileName,
 	                                    deterministicRandom()->randomUniqueID());
 }
 
+Future<Version> FileBackupAgent::restore(Database cx,
```

Comment:
Keep the original API prototype and redirect to new restore API with beginVersions